classes/system/galera/server/database/aodh.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
```

Comment:
I'm confused which of this class and system.mysql.client.database.aodh is used...